aws-datastore/src/main/java/com/amplifyframework/datastore/appsync/AppSync.java

```diff
@@ -112,7 +112,7 @@ <T extends Model> Cancelable create(
     <T extends Model> Cancelable update(
             @NonNull T model,
             @NonNull ModelSchema modelSchema,
-            @NonNull Integer version,
+            @Nullable Integer version,
```

Comment:
Just noting that this changes a public contract. However, these are annotations that may throw warnings, but the change shouldn't explicitly break anything. This filed is really likely only public due to Java interface rules